apps/emqx_bridge/src/emqx_bridge_api.erl

```diff
@@ -555,8 +561,8 @@ schema("/bridges_probe") ->
             case emqx_bridge_resource:create_dry_run(ConnType, maps:remove(<<"type">>, Params)) of
                 ok ->
                     {204};
-                {error, Error} ->
-                    {400, error_msg('TEST_FAILED', Error)}
+                {error, Reason} when not is_tuple(Reason); element(1, Reason) =/= 'exit' ->
+                    {400, error_msg('TEST_FAILED', to_hr_reason(Reason))}
```

Comment:
🤔 Won't this fail with 500 and case clause on `{error, {exit, ...}}`? Is this intentional?

Reply:
Yes it will and it's by intention. Don't think it's worth its own handler. Yes, the error will be one level more nested but it shouldn't happen in the first place, so if we see it, we'll fix whatever caused it and we have less code to maintain. 🤷